### ISSUE TITLE
Incorrect declaration of task property

### ISSUE THREAD

**Issue by lptr** [ysb33r/ivypot-gradle-plugin#26]
This code has no effect:

```groovy
        inputs.properties.put('project configurations', {
            this.projectConfigurations
        })
```

https://github.com/ysb33r/ivypot-gradle-plugin/blob/master/src/main/groovy/org/ysb33r/gradle/ivypot/OfflineRepositorySync.groovy#L60-L62

Please use this instead:

```groovy
        inputs.property('project configurations', {
            this.projectConfigurations
        })
```

**Comment by ysb33r** [comment]
Nice find.

**Comment by ysb33r** [comment]
Will be fixed in 0.7.
